### ISSUE TITLE
Is coco_detection.py line 45 an error?

### ISSUE THREAD

**Issue by eguoguo321** [open-mmlab/mmdetection#4170]
Thanks for your error report and we appreciate it a lot.

**Checklist**
yes,it seems like no one see it

**Describe the bug**
IT seems like an little error or not?
in /mmdetection/configs/_base_/datasets/coco_detection.py line 45 and line 46
```
test=dict(
        type=dataset_type,
        ann_file=data_root + 'annotations/instances_val2017.json',
        img_prefix=data_root + 'val2017/',
        pipeline=test_pipeline))
```

**Reproduction**
I use retinanet to train my own model ,and I saw code in  coco_detection.py just like a error.

**Environment**

I use docker which U provided,and I added some useful thing

**Error traceback**

**Bug fix**

I think that should be 

```
    test=dict(
        type=dataset_type,
        ann_file=data_root + 'annotations/instances_test2017.json',
        img_prefix=data_root + 'test2017/',
        pipeline=test_pipeline))
```

**Comment by ZwwWayne** [comment]
Hi @eguoguo321 ,
This is because usually we still use the validation set of COCO data during testing unless we want to submit test-dev or test set results to the evaluation server. Therefore, by default we still use `instances_val2017.json`.
